README.md

```diff
@@ -21,6 +21,7 @@ mv $CMSSW_BASE/external/$SCRAM_ARCH/data/RecoEgamma/ElectronIdentification/data/
 ## Add the BParkingNano package and build everything
 
 ```
+git cms-addpkg PhysicsTools/NanoAOD
```

Comment:
why NanoAOD needed explicitly?
Also, since you are modifying the README can I ask to update the instructions  "To run on a test file"

Reply:
sorry just read with more attention your comments :) NanoAOD is needed explicitly!